browser/src/layer/marker/TextInput.js

```diff
@@ -769,11 +789,7 @@ L.TextInput = L.Layer.extend({
 		// MSIE/Edge cannot compare a string to "\n" for whatever reason,
 		// so compare charcode as well
 		if (text === '\n' || (text.length === 1 && text.charCodeAt(0) === 13)) {
-			// The composition messages doesn't play well with just a line break,
```

Comment:
I guess we should test this with an input method - like SCIM on Linux - any chance of setting that up ? you can type eg. 'nippon' and get the character(s) for Japan in the Japanse locale

Reply:
Oh well, I have no experience with SCIM or alike input method. I'll add it to my todo list. Anyway the code here is the same as before just moved into `_sendNewLineEvent()`

Reply:
SCIM tested: it works as with <textarea>